bgpd/bgp_fsm.c

```diff
@@ -456,6 +456,8 @@ void bgp_timer_set(struct peer *peer)
 		bgp_keepalives_off(peer);
 		BGP_TIMER_OFF(peer->t_routeadv);
 		break;
+	case BGP_STATUS_MAX:
```

Comment:
Shouldn't we have an error log or assert here ?

Reply:
fixed

Reply:
fixed